internal/characters/dori/attack.go

```diff
@@ -0,0 +1,61 @@
+package dori
+
+import (
+	"fmt"
+
+	"github.com/genshinsim/gcsim/internal/frames"
+	"github.com/genshinsim/gcsim/pkg/core/action"
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/combat"
+)
+
+var (
+	attackFrames          [][]int
+	attackHitmarks        = [][]int{{27}, {19, 33}, {60}}
+	attackHitlagHaltFrame = [][]float64{{0.1}, {0, 0}, {0.08}}
```

Comment:
you need an extra array for CanBeDefenseHalted with [][]bool{{true}, {false, false}, {true}}, because rn you would have hitlag on N2-1 and N2-2. see sayu's code for example